Source/Scene/Expression.js

```diff
@@ -917,19 +907,18 @@ define([
         }
 
         var member = this._right.evaluate(frameState, feature);
-        if (property instanceof Color) {
-            // Color components may be accessed with .x, .y, .z, .w and implicitly with .red, .green, .blue, .alpha
-            if (member === 'x') {
-                return property.red;
-            } else if (member === 'y') {
-                return property.green;
-            } else if (member === 'z') {
-                return property.blue;
-            } else if (member === 'w') {
-                return property.alpha;
+        if ((property instanceof Cartesian2) || (property instanceof Cartesian3) || (property instanceof Cartesian4)) {
+            // Vector components may be accessed with .red, .green, .blue, .alpha and implicitly with .x, .y, .z, .w
```

Comment:
Like how glsl can access components of a vector with `.r`, `.g`, `.b`, `.a`, I kept in the `.red`, `.green`, `.blue`, `.alpha`. Should they be shortened to `.r`, etc instead, or should this be removed?